src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift

```diff
@@ -433,6 +434,18 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 		])
 	}
 
+	private func faqLinkText(tintColor: UIColor = .enaColor(for: .textTint)) -> NSAttributedString {
```

Comment:
The link seems to have the wrong color its a dark Blue instead of our tint color.

Reply:
I never applied the tint color. Fixed and screenshots are updated.